addons/mrp/static/src/scss/mrp_workorder_calendar.scss

```diff
@@ -0,0 +1,4 @@
+.o_cw_popover_delete
+{
+  visibility: hidden;
```

Comment:
wouldn't it be better to adapt CalendarView.event.popover to not include the delete button at all if not allowed? inside a t-if

Reply:
@sle-odoo Done.